en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -894,12 +910,19 @@ class MyVersionedObject extends DataObject
 }
 ```
 
-### Configure frontend asset building
+#### Configure frontend asset building {#history-viewer-js}
 
-If you haven't already configured frontend asset building for your project, you will need to configure some basic
-packages to be built via webpack in order to enable history viewer functionality. If you have this configured for
-your project already, ensure you have the `react-apollo` and `graphql-tag` libraries in your `package.json`
-requirements, and skip this section.
+If you haven't already configured frontend asset (javascript/css) building for your project, you will need to configure some basic
+packages to be built in order to enable history viewer functionality. This section includes a very basic webpack configuration which uses [@silverstripe/webpack-config](https://www.npmjs.com/package/@silverstripe/webpack-config).
+
+[hint]
+If you have this configured for your project already, ensure you have the `react-apollo` and `graphql-tag` libraries in your `package.json`
+requirements (with the appropriate version constraints from below), and skip this section.
+[/hint]
+
+[notice]
+Using `@silverstripe/webpack-config` will keep your transpiled bundle size smaller and ensure you are using the correct versions of `react-apollo` and `graphql-tag`, as these will automatically be added as [webpack externals](https://webpack.js.org/configuration/externals/). If you are not using that npm package, it is very important you use the correct versions of those dependencies.
+[/notice]
 
```

Comment:
Perhaps we could clarify that the libraries should be installed as `dependencies` somewhere in "hint" or "notice" section. Just in case if the user skips the `package.json` snippet

Reply:
Ahhh, but they don't have to be - that's the beauty of webpack externals.